js/components/chart-base.tsx

```diff
@@ -165,7 +165,9 @@ class ChartBase extends React.Component<IChartBaseProps, null> {
     }
 
     componentWillReceiveProps(nextProps) {
-        if (this.props.isLoading && !nextProps.isLoading)
+        const isEqual = require('react-fast-compare');
+
+        if ((this.props.isLoading && !nextProps.isLoading) || isEqual(this.props.objectData, nextProps.objectData))
```

Comment:
Why does it reload if the new objectdata is equal, I thought this would reload when the new props are different?

Reply:
Could you also change add braces to ifs and statements.
https://palantir.github.io/tslint/rules/curly/
Whenever we change these old bootstrap files, we like to pretty them up.